src/manual/MoBiToolboxForR-package.Rd

```diff
@@ -12,8 +12,8 @@ This package provides functions to access a MoBi
 \tabular{ll}{
 Package: \tab MoBiToolboxForR\cr
 Type: \tab Package\cr
-Version: \tab 6.3.1\cr
-Date: \tab 2016-07-20\cr
+Version: \tab 7.3.0\cr
+Date: \tab 2018-30-01\cr
```

Comment:
Could not that be read from a file generated during build? This would be so much nicer than always having to adapt it by hand

Reply:
Maybe